templates/default/push_tiles.py.erb

```
@@ -128,14 +130,11 @@ if __name__ == "__main__":
   #for each arg
   for f in sys.argv[1:]:
     #check all the files are there
-    try:
-      if stat.S_ISREG(os.stat(f).st_mode) == False:
-        raise
-      #push them to s3
-      else:
-        push_to_s3(f, '<%= node[:valhalla][:bucket] %>', '<%= node[:valhalla][:bucket_dir] %>')
-    except:
```

this was masking exceptions raised in the push_to_s3 function